data/json/monsters.json

```diff
@@ -1864,6 +1864,44 @@
     "death_function": [ "BROKEN" ],
     "flags": [ "SEES", "FLIES", "ELECTRONIC", "NO_BREATHE", "NOHEAD", "PRIORITIZE_TARGETS" ]
   },
+  {
+    "id": "mon_zombie_flamer",
+    "type": "MONSTER",
+    "name": "zombie flamer",
```

Comment:
I'd recommend changing the name - "flamer" in American English typically refers to a flamboyant gay man